bitcoin/secp256k1_main.py

```diff
@@ -239,23 +236,17 @@ def multiply(s, pub, usehex, rawpub=True):
     of the scalar s.
     ('raw' options passed in)
     '''
-    newpub = secp256k1.PublicKey(pub, raw=rawpub, ctx=ctx)
-    #see note to "tweak_mul" function in podle.py
-    res = secp256k1._tweak_public(newpub,
-                                   secp256k1.lib.secp256k1_ec_pubkey_tweak_mul,
-                                   s)
-    return res.serialize()
+    res = coincurve.PublicKey(pub)
+    res.multiply(s, update=True)
```

Comment:
I think ...:

using update=True is going to multiply the point being called, and that's sometimes not what's wanted.

For example in the PoDLE construction, in the function `podle.PoDLE.generate_podle()` we have lines like:

    KJ = tweak_mul(J, k)

This will I believe update J to k*J. As it happens I don't think it'll matter in that specific function. `bitcoin.multiply` is also called in `configure.donation_address`, and at a quick glance, it looks like that does get screwed up (because `reusable_donation_pubkey` is referred to again after the update).